web/src/api/openapi-server/auth.ts

```
@@ -245,6 +276,34 @@ export const authEmailPasswordSignin = async (
   );
 };
 
+/**
+ * Request password reset email to be sent to the specified email address.
+
+ */
+export type authPasswordResetRequestEmailResponse = {
+  data: void;
```

Biome

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)